src/OpenTelemetry.Exporter.OneCollector/Logs/OneCollectorLogExportProcessorBuilder.cs

```diff
@@ -202,7 +202,9 @@ internal BaseProcessor<LogRecord> BuildProcessor()
 
     private OneCollectorExporter<LogRecord> BuildExporter()
     {
+#pragma warning disable CA2000 // Dispose objects before losing scope
```

Comment:
nit: You could try renaming this method "CreateExporter" it _may_ resolve CA2000. It tries to use names of things to detect factory patterns where it is expected that something will be returned.

Reply:
I tried that but it didn't help.